### PR TITLE
Use disguised flag

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/760

cc @GuillaumeGomez, @sdroege 

We lose all Debug for disguised and test checks for "GdkAtom" and "GIConv"

### CODE REVIEW

File: glib-sys/src/lib.rs [hunk 1]
```diff
@@ -1055,24 +1055,14 @@ impl ::std::fmt::Debug for GArray {
...
-pub struct GBookmarkFile(c_void);
+pub struct _GBookmarkFile(c_void);
 
-impl ::std::fmt::Debug for GBookmarkFile {
-    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        f.debug_struct(&format!("GBookmarkFile @ {:?}", self as *const _))
-         .finish()
-    }
-}
+pub type GBookmarkFile = *mut _GBookmarkFile;
```
Comment (sdroege) [inline glib-sys/src/lib.rs]: Why this representation change?

Comment (EPashkin) [reply glib-sys/src/lib.rs]: This current representation for disguised types,
made for GdkAtom and GIConv

Comment (GuillaumeGomez) [reply glib-sys/src/lib.rs]: Good for you @sdroege ?

Comment (sdroege) [reply glib-sys/src/lib.rs]: Yes, was just curious if this was intentional and why.

**Pull request merged**